docs/release-notes/saros-i_0.1.0.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Saros/I 0.1.0 Release Notes
+---
+
+This is the first alpha release of Saros/I, so expect it to be a bit rough around the edges.
+There are still some [restrictions](#restrictions) that apply to the usage and some basic [features are still missing](#missing-features).
+This release is not compatible with any other Saros plugin/version (like Saros/E).
```

Comment:
If you give the plugin the same version number as Saros/E it would be compatible. There is no mechanism beside the version number comparison, i.e there is no additional information to dertermine compability.

Reply:
Yes, I am aware of this. But, as there is currently no other Saros version with the release version 0.1.0, this should be fine for the first release (as the user would have to actively try to mess this up). Later on, we can modify the version compatibility logic to differentiate between IDE versions (as I suggested in the discussion of [PR 465](https://github.com/saros-project/saros/pull/465#issuecomment-470280748).